src/cascadia/TerminalSettingsModel/KeyChordSerialization.cpp

```diff
@@ -5,6 +5,7 @@
 #include "KeyChordSerialization.h"
 #include "KeyChordSerialization.g.cpp"
 
+#include <charconv>

```

Comment:
huh which thing required charconv again?

Reply:
I initially tried to convert it from wchar_t to char, but I was so annoyed by that, that I `stroul` which even had support for `0x` prefixes (nice!). But that one again doesn't work with non-null-terminating strings. This experience and utter lack of support for wchar_t number parsing was so infuriating for me I just built my own function in the end. 😄